app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeViewModel.kt

```diff
@@ -16,4 +23,20 @@ class HomeViewModel : ViewModel() {
     val titleText: LiveData<String> = _titleText
     val descriptionText: LiveData<String> = _descriptionText
 
+    // TODO remove this when we can research the cards and open them in the single card fragment
+    // TODO the things to remove are: localDatabase, cards, generateCards, getRandomCard, the button
+    var localDatabase: AppDatabase? = null
+
+    private val cards = generateCards()
+
+    fun addRandomCard() {
+        val toInsert = cards.random()
+        viewModelScope.launch {
+            localDatabase?.magicCardDao()
```

Comment:
Why the two `?` here? Why would your database or DAO maybe be null, and if they are, should this really be ignored? (I realize this is a temporary feature, but it looks odd)